openshift-knative-operator/cmd/operator/kodata/ingress/1.8/0-kourier.yaml

```diff
@@ -0,0 +1,539 @@
+---
```

Comment:
It would be nice to change the name of the file to `0-net-kourier` in the future.
Also change the policy one to `0-net-istio-policies.yaml` or something similar to that for readability reasons.

Reply:
Agreed. Do we need an issue to track that? Also we dropped the indexes in serving midstream. Either way, we should align. /cc @nak3

Reply:
Yeah, no problem for the change.

Reply:
I will create a jira ;)